test/performance/const_division.cpp

```diff
@@ -40,18 +40,33 @@ bool test(int w, bool div) {
         for (int x = 0; x < input.width(); x++) {
             uint32_t bits = (uint32_t)rng();
             input(x, y) = (T)bits;
+            // Round-to-zero faults on zero denominators
+            if (round_to_zero && (input(x, y) == 0)) {
+                input(x, y) = 1;
+            }
         }
     }
 
     if (div) {
-        // Test div
-        f(x, y) = input(x, y) / cast<T>(y + min_val);
+        if (round_to_zero) {
+            // Test div
+            f(x, y) = div_round_to_zero(input(x, y), cast<T>(y + min_val));
 
-        // Reference good version
-        g(x, y) = input(x, y) / cast<T>(y + min_val);
+            // Reference good version
```

Comment:
This looks identical to the case just above, are they supposed to be identical?

Reply:
Yes, they have different schedules which turn the denominator into a constant in one case but not the other.

Reply:
(I'll add a comment)